rpa-manager/modules/ROOT/pages/processoperations-upcomingprocesschanges.adoc

```diff
@@ -0,0 +1,92 @@
+
+To view the changes:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. View the descriptions in the table.
+. To compare the deployed version with the latest version of an activity, click the *Show Changes* icon image:show-icon.png[eye symbol,1.5%,1.5%] in the table row of the edited activity.
+
+[[map-bpmn-elements-to-process-activities]]
+== Map BPMN Elements to Process Activities
+
+If a process contains the same activity more than once, use your browser's developer tools to find out which BPMN element contains the referenced activity revision:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. Right-click on a process name, open the project in another tab and open the *Automation* panel there.
+. In the *Automation* panel, right-click on one of the possible activities and choose *Inspect* from the browser's context menu. 
+
+The *Elements* panel of the browser shows the highlighted source code of the element. 
+
+To map this element to an element shown in the *Upcoming Process Changes* table:
+
+. In the *Elements* panel, look up the text next to *data-element-id=* contained in the source code belonging to the highlighted text.
+. Switch back to the *Upcoming Process Changes* tab.
+. Compare the *data-element-id* with the *BPMN Element* id in the list of all process changes.
+
+Matching activities have the same id.
+ 
+image::rpa_processOperations_UpcomingProcessChanges_WhichBPMNElement.png[Screenshots of the Upcoming Process Changes table and the Automation Panel's developer tools elements view with an element highlighted. An arrow shows matching IDs.]
+
+[[deploy-the-latest-version-of-an-activity]]
+== Deploy the Latest Version of an Activity
+
+To ensure that the processes use an appropriate version of a referenced activity, you can deploy the latest version to processes in the design, build, or test phase.
+
+Before you deploy the latest version of an activity:
+
+* <<view-upcoming-process-changes, View the upcoming process changes>>.
+* If a process contains the same activity more than once, <<map-bpmn-elements-to-process-activities, ensure that you know which activity to select>>.
+* If processes containing an upcoming change are in the produtcion phase, transfer the processes to the test phase to deploy the changed activity.
+
+To deploy the latest version of an activity:
+
+. Open the *Upcoming Process Changes* view in the *Process Operations* module.
+. Select the activities to deploy.
+. Click *Deploy*.
+
+Test the updated processes before transfering them to production phase.
+
+== Deploy A Prior Version of an Activity
+
+To ensure that a process uses an appropriate version of a referenced activity or to undo a prior deployment, you can change the version of a referenced activity in the RPA Builder:
+
+. Transfer the process to the build phase.
+. Open the process in the RPA Builder.
```

Comment:
```suggestion
. Open the process in RPA Builder.
```

Reply:
Changed every occurence of
the RPA Manager/Builder/Recorder in the RPA Manager docs
(and implemented som vale suggestions)